conf/modules.config

```diff
@@ -59,7 +59,7 @@ process {
 
 process {
     withName: '.*PREPARE_REFERENCES:BWAMEM2_INDEX_GENOME' {
-        ext.when = {!params.bwamem2_index && params.aligner == "bwamem2"}
+        ext.when = {!params.bwamem2_index}
```

Comment:
Does that mean that this will always run, even when aligner == sentieon?

Reply:
I made this change because the mitochondrial subworkflow uses bwamem2. Ideally we should make changes there so that it works with sentieon, but that's going to take a while so I made this change as a temporary fix.

Reply:
@Gwennid I have addressed this issue now using my first suggestion. MT subworkflow can run using either bwamem2 or sentieon.
Adding a switch would be nice as well, but I will do that in a separate PR 😄